utils/oauth-app-detail.sh

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env bash
```

Comment:
Skripty na nastavenie oauth som zobral z OneDrive komponent a trocha opravil drobne chyby (ak bola medzera v nejakej premennej prostredia, tak to padalo).


https://github.com/keboola/ex-onedrive/tree/master/utils